cfpb/consumerfinance.gov
# Update so that margins auto by default and 170 or 270 classes apply regardless of screen size

Fixes inset images to be correct size and centered on smaller screens

## Changes

- Updates to inset.less


## How to test this PR

1. Visit the following link while running locally: 
http://localhost:8000/start-small-save-up/start-saving/how-to-use-your-tax-refund-to-build-your-emergency-funds/
2. Using Chrome tools change the size dynamically to make sure the page images still look good.



## Checklist



  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
